WeatherMonitoring.Tests/Tests/BotFactoryTests.cs

```diff
@@ -0,0 +1,41 @@
+using FluentAssertions;
+using WeatherMonitoring.Enums;
+using WeatherMonitoring.WeatherBots;
+using WeatherMonitoring.WeatherBots.BotConfigurations;
+
+public class BotFactoryTests
+{
+    [Fact]
+    public void CreateBots_ReturnsOnlyEnabledBots()
```

Comment:
Test case's names should follow the convention of MethodName_Scenario_Expectation.

An example

```suggestion
    public void CreateBots_ShouldReturnBots_WhenTheyAreEnabled()
```